matrix_benchmarking/store/simple.py

```diff
@@ -62,6 +62,7 @@ def _parse_directory(results_dir, expe, dirname):
                         continue
 
                     import_settings[key] = value
+                    import_settings['lts'] = 0
```

Comment:
can you remove this?
I prefer not to have it at the moment, we'll decide later if that's mandatory or not for having a proper sorting

Reply:
ping, this should be removed